dev/AnimatedIcon/AnimatedIcon.cpp

```diff
@@ -0,0 +1,521 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "common.h"
+#include "AnimatedIcon.h"
+#include "RuntimeProfiler.h"
+#include "ResourceAccessor.h"
+#include <AnimatedVisuals\ProgressRingDeterminate.h>
+#include <AnimatedIconTestHooks.h>
+using namespace AnimatedVisuals;
+
+AnimatedIcon::AnimatedIcon()
+{
+    __RP_Marker_ClassById(RuntimeProfiler::ProfId_AnimatedIcon);
+    m_progressPropertySet = winrt::Window::Current().Compositor().CreatePropertySet();
+    m_progressPropertySet.InsertScalar(L"Progress", 0);
```

Comment:
nit: we use constant elsewhere. Example:
        m_expressionAnimationSources.InsertScalar(s_zoomFactorSourcePropertyName, 0.0f);